Assets/Scripts/Bullseye.cs

```diff
@@ -21,4 +23,16 @@ public bool HasCollidedWith(Vector3 pos)
     {
         return Vector3.Distance(pos, transform.position) < (transform.localScale.x / 2);
     }
+
+    private void OnTriggerEnter(Collider other)
```

Comment:
please do not use OnTriggerEnter inside the bullseye target as it is technically possible to collide with the bullseye without hitting it's graphics given that it uses a box collider (there are no cylinder collider in unity)

Instead you can fetch the time of the first finished racer in the racer manager by checking if it's the first time since the start of the scene that a racer has been removed. If you need help finding where in the code to do that let me know